core/src/run.rs

```diff
@@ -134,13 +134,15 @@ impl RunStatus {
 pub enum RunType {
     Deploy,
     Local,
+    Execute,
 }
 
 impl ToString for RunType {
     fn to_string(&self) -> String {
         match self {
             RunType::Deploy => "deploy".to_string(),
             RunType::Local => "local".to_string(),
+            RunType::Execute => "execute".to_string(),
```

Comment:
Went with "execute", felt like "run" was ambiguous and "use" too much tied to the frontend

Reply:
Yes that looks good to me. Maybe we'll migrate "Local" to "Design" in the future.